test/BasicNetworkTest.template.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 SYMBOL
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Network } from '@core';
+import { expect } from 'chai';
+
+export const AssertNetwork = (network: Network, expectedName: string, expectedIdentifier: number): void => {
+    it('correct predefined networks are registered', () => {
+        // Assert:
+        expect(network.name).to.be.equal(expectedName);
+        expect(network.identifier).to.be.equal(expectedIdentifier);
+    });
+};
+
+export const BasicNetworkTester = (networks: Network[], name: string, identifier: number): void => {
+    it('can find well known network by name', () => {
+        // Act:
+        const network = Network.findByName(networks, name);
+
+        if (network) {
+            AssertNetwork(network, name, identifier);
+        }
```

Comment:
// Assert:
i don't think you want this to be conditional, else test will pass if findByName returns undefined